src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Ball {
+
+    private String color;
+    private int number;
+
+    public int getNumber() {
+        return number;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    @Override
+    public String toString() {
+        return super.toString();
+    }
```

Comment:
The `toString()` method should be overridden to provide a meaningful representation of the Ball object. Currently, it's calling `super.toString()` which will return a string that is the representation of the object's hash code in hexadecimal form, which is not informative. Consider returning a string that includes the ball's color and number.

Reply:
fix comment